crates/store/src/server/api.rs

```diff
@@ -141,7 +142,15 @@ impl api_server::Api for StoreApi {
             })
             .collect();
 
-        let transactions = state.transactions.into_iter().map(Into::into).collect();
+        let transactions = state
+            .transactions
+            .into_iter()
+            .map(|transaction_info| TransactionSummary {
```

Comment:
minor nit: variable name could probably be `transaction_summary` or just `transaction`.

Reply:
renamed to `transaction_summary`